src/tests/routes/test_alerts.py

```diff
@@ -89,11 +91,12 @@ async def init_test_db(monkeypatch, test_db):
 @pytest.mark.parametrize(
     "access_idx, alert_id, status_code, status_details",
     [
-        [0, 1, 401, "Permission denied"],
+        [0, 1, 200, None],
         [1, 1, 200, None],
         [2, 1, 401, "Permission denied"],
         [1, 999, 404, "Entry not found"],
         [1, 0, 422, None],
+        [4, 1, 401, "You can't access this ressource"],
```

Comment:
A "Permission denied" would be better to unify error messages

Reply:
HMm, I personnally think that it would be helpful for the user to let him know that he is allowed to use that route but that he should check its input

Reply:
Perhaps let's make it a more detailed message then, because I understand it the same way as "Permission denied" (which can be applied to route or sub entry getting)

What I mean is: the route will fail anyway, the user won't know where and in one case that person will get "Permission denied" and another "You can't access this resource". I would understand that they meant there is a difference but I wouldn't have any clue what it is :sweat_smile: